src/components/MapView/index.tsx

```diff
@@ -76,6 +97,50 @@ const dateSupportLayerTypes: Array<LayerType['type']> = [
   'wms',
 ];
 
+const getActiveFeatureInfoLayers = (map: Map): WMSLayerProps[] | undefined => {
```

Comment:
Maybe move these functions to a util file? Up to you :+1:

Reply:
Included in e9b782b72e35d52d6ec74782d1dac7aee42d3729